pkg/chunk/cache/background_test.go

```diff
@@ -13,14 +13,14 @@ type mockCache struct {
 	cache map[string][]byte
 }
 
-func (m *mockCache) StoreChunk(_ context.Context, key string, buf []byte) error {
+func (m *mockCache) Store(_ context.Context, key string, buf []byte) error {
 	m.Lock()
 	defer m.Unlock()
 	m.cache[key] = buf
 	return nil
 }
 
-func (m *mockCache) FetchChunkData(ctx context.Context, keys []string) (found []string, bufs [][]byte, missing []string, err error) {
+func (m *mockCache) Fetch(ctx context.Context, keys []string) (found []string, bufs [][]byte, missing []string, err error) {
```

Comment:
Can we avoid renaming like this mixed in with new functionality?  Makes it hard to review.

Reply:
Well, I've thought about it. If I didn't rename it, it'll mean creating a new interface and copying most of the `Tier*` code. Or using misnamed functions for index too.

I've put all the renaming in the first commit, to help with the review. If you could ignore the first commit using the GH UI, it should make it easier.

Reply:
Fair enough I think.